server/protocols/jmap-draft-integration-testing/jmap-draft-integration-testing-common/src/test/java/org/apache/james/jmap/draft/methods/integration/GetMessageListMethodTest.java

```diff
@@ -966,7 +966,6 @@ public void getMessageListShouldReturnAllMessagesOfCurrentUserOnlyWhenMultipleMa
         mailboxProbe.createMailbox(MailboxConstants.USER_NAMESPACE, ALICE.asString(), "mailbox2");
         ComposedMessageId message2 = mailboxProbe.appendMessage(ALICE.asString(), MailboxPath.forUser(ALICE, "mailbox2"),
                 new ByteArrayInputStream("Subject: test2\r\n\r\ntestmail".getBytes()), new Date(), false, new Flags());
-        await();
```

Comment:
why?

Reply:
Because we can only await once after all the data to be appended.

Reply:
please put that in the commit description please